bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/dialogs/SearchPattern.java

```diff
@@ -52,7 +56,9 @@ public class SearchPattern {
 	/**
 	 * Match rule: The search pattern matches the search result only if cases are
 	 * the same. Can be combined with previous rules, e.g. {@link #RULE_EXACT_MATCH}
-	 * | {@link #RULE_CASE_SENSITIVE}
+	 * | {@link #RULE_CASE_SENSITIVE}.
+	 * <p>
+	 * TODO The actual code seems to ignore this flag when performing searches.
```

Comment:
I've noticed this when going through the code, I'm not sure what to think about it / what to do about it. IMHO luckily nobody probably cares about this case-sensitivity flag in their client code...?

Reply:
This TODO shouldn't be in Java doc, but a regular comment (accessible only to anyone who reads the code).

Reply:
OK, done. I thought it wouldn't be bad to also let the clients know directly, but OK, never mind.